<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintOptionDialog</class>
 <widget class="QDialog" name="PrintOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="typeGroupBox">
     <property name="title">
      <string>Print Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="printActiveBoard">
        <property name="text">
         <string>Print current board.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="printActiveBranch">
        <property name="text">
         <string>Print active branch.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="printActiveBranchN">
          <property name="text">
           <string>Print active branch, Number of moves per page.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="activeBranchMovesPerPage">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="printAllBranches">
        <property name="text">
         <string>Print all branches.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="printAllBranchesN">
          <property name="text">
           <string>Print all branches, Number of moves per page.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="allBranchMovesPerPage">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionBox">
     <property name="title">
      <string>Print Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="showCoordinate">
        <property name="text">
         <string>Show Coordinate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="includeComments">
        <property name="text">
         <string>Includes Comments</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QFontComboBox" name="fontComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="fontSizeComboBox">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="headerFooterGroupBox">
     <property name="title">
      <string>Header and Footer</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="headerLeftLabel">
        <property name="text">
         <string>Header(Left)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="headerLeftEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="headerCenterLabel">
        <property name="text">
         <string>Header(Center)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="headerCenterEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="headerRightLabel">
        <property name="text">
         <string>Header(Right)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="headerRightEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="footerLeftLabel">
        <property name="text">
         <string>Footer(Left)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="footerLeftEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="footerCenterLabel">
        <property name="text">
         <string>Footer(Center)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="footerCenterEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="footerRightLabel">
        <property name="text">
         <string>Footer(Right)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="footerRightEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>printActiveBoard</tabstop>
  <tabstop>printActiveBranch</tabstop>
  <tabstop>printActiveBranchN</tabstop>
  <tabstop>activeBranchMovesPerPage</tabstop>
  <tabstop>printAllBranches</tabstop>
  <tabstop>printAllBranchesN</tabstop>
  <tabstop>allBranchMovesPerPage</tabstop>
  <tabstop>showCoordinate</tabstop>
  <tabstop>includeComments</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>fontSizeComboBox</tabstop>
  <tabstop>headerLeftEdit</tabstop>
  <tabstop>headerCenterEdit</tabstop>
  <tabstop>headerRightEdit</tabstop>
  <tabstop>footerLeftEdit</tabstop>
  <tabstop>footerCenterEdit</tabstop>
  <tabstop>footerRightEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrintOptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrintOptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
